.github/workflows/backport-dispatch.yaml

```
@@ -0,0 +1,34 @@
+name: Automated Backporting via Dispatch
+
+on:
+  workflow_dispatch:
+    inputs:
+      targetBranch:
+        description: 'Release (22.0, 24.0, ...)'
+        required: true
+        type: string
+        default: "24.0"
+      pullRequest:
+        description: 'Pull request (just the ID)'
```

Should be number, PRs have both a number (28087) and an ID (PR_kwDOAKnDVc5qMnnG)

```suggestion
        description: 'Pull request number'
```